client/azure_client_set.go

```diff
@@ -209,7 +209,7 @@ func newPublicIPAddressesClient(authorizer autorest.Authorizer, subscriptionID,
 	return &client, nil
 }
 
-func newSecurityRulesClient(authorizer autorest.Authorizer, subscriptionID, partnerID string) (interface{}, error) {
+func newSecurityGroupsClient(authorizer autorest.Authorizer, subscriptionID, partnerID string) (*network.SecurityRulesClient, error) {
```

Comment:
This was a typo right? Let's keep the fix.

Reply:
done